packages/typespec-ts/package.json

```diff
@@ -64,7 +70,8 @@
     "@azure/core-rest-pipeline": "^1.12.0",
     "@azure/logger": "^1.0.4",
     "@azure/core-util": "^1.4.0",
-    "eslint-plugin-require-extensions": "0.1.3"
+    "eslint-plugin-require-extensions": "0.1.3",
+    "@typespec/ts-http-runtime": "1.0.0-alpha.20231103.1"
```

Comment:
@joheredi Do we have a plan to review a preview version for runtime lib before Nov Demo?